src/subdomains/generic/kyc/dto/sum-sub.dto.ts

```diff
@@ -0,0 +1,184 @@
+import { IdentShortResult } from './input/ident-result.dto';
+
+export interface SumSubResult {
+  applicantId?: string;
+  applicantActionId?: string;
+  applicantType?: ApplicantType;
+  inspectionId?: string;
+  correlationId?: string;
+  externalUserId?: string;
+  externalApplicantActionId?: string;
+  levelName?: string;
+  previousLevelName?: string;
+  type?: WebhookType;
+  reviewResult?: {
+    reviewAnswer: ReviewAnswer;
+    moderationComment?: string;
+    clientComment?: string;
+    rejectLabels?: RejectionLabels[];
+    reviewRejectType?: ReviewRejectType;
+    buttonIds?: string[];
+  };
+  reviewStatus?: string;
+  videoIdentReviewStatus?: string;
+  createdAt: Date;
+  createdAtMs?: Date;
+  sandboxMode?: boolean;
+  clientId?: string;
+  reviewMode?: string;
+}
+
+export interface SumSubApplicantDocuments {
+  IDENTITY: {
+    reviewResult: {
+      reviewAnswer: ReviewAnswer;
+    };
+    country: string;
+    idDocType: 'ID_CARD';
+    imageIds: [861042510, 1897370144];
+    imageReviewResults: {
+      '861042510': {
+        reviewAnswer: 'GREEN';
+      };
+      '1897370144': {
+        reviewAnswer: 'GREEN';
+      };
+    };
+    forbidden: false;
+    partialCompletion: null;
+    stepStatuses: null;
+    imageStatuses: [];
+  };
+  SELFIE: {
+    reviewResult: {
+      reviewAnswer: 'GREEN';
+    };
+    country: 'ZAF';
+    idDocType: 'SELFIE';
+    imageIds: [325528857];
+    imageReviewResults: {
+      '325528857': {
+        reviewAnswer: 'GREEN';
+      };
+    };
+    forbidden: false;
+    partialCompletion: null;
+    stepStatuses: null;
+    imageStatuses: [];
+  };
+}
```

Comment:
Fix types (remove fixed string values, nulls, booleans, ...)

Reply:
Is not in use. Will remove it was from some testing.

Reply:
Ok 👍